.github/workflows/checks-windows.yml

```
@@ -20,7 +20,7 @@ permissions: read-all
 jobs:
   fuzz:
     name: Fuzz
-    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@main
+    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@fix-fuzz-job-windows
```

This change **must** be reverted before this is merged.

```suggestion
    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@main
```